### PR TITLE
[MAR-2475] seller dashboard

### PR DESCRIPTION
**Pull request by malmas18**

### CODE REVIEW

File: apps/marketplace/components/SellerDashboard/SellerDashboard.scss [hunk 1]
```diff
@@ -115,6 +125,28 @@ table.resultListing td, table.resultListing th {
...
+
+}
+.notification .count {
+  color: white;
+  width: 24px;
+  font-size: 1rem;
+  top: 0px;
+  left: 1px;
+  position: absolute;
+  text-align: center;
```
Comment (tobiasbrown) [inline apps/marketplace/components/SellerDashboard/SellerDashboard.scss]: The circle and count look a little off to me.  If you move `border-radius: 50%` from `circle` to `count` we should be able to delete the circle div from the markup.  Do you think that looks better (only for two digit numbers)?

File: apps/marketplace/components/SellerDashboard/Messages.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+    if (!loading && !errors) {
+      const difference = differenceInSeconds(new Date(), loadedAt)
+      if (difference > 60) {
+        this.props.loadData()
+      }
+    }
+  }
+
+  formatMessage = (item, row) => {
```
[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/SellerDashboard/Messages.js]: deleted component

File: apps/marketplace/components/SellerDashboard/Services.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+                      >
+                        View criteria
+                      </a>
+                      <a
+                        target="_blank"
+                        href={`https://marketplace1.zendesk.com/hc/en-gb/articles/360000556476${this.serviceIdToHash(
+                          item
+                        )}`}
+                      >
+                        View rates
```
[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/SellerDashboard/Services.js]: yep, following design for now

File: apps/marketplace/components/SellerDashboard/SellerDashboard.scss (hunk 1 above)
Comment (malmas18) [reply apps/marketplace/components/SellerDashboard/SellerDashboard.scss]: fixed! 👍

File: apps/marketplace/components/SellerDashboard/SellerDashboard.scss [hunk 4]
```diff
@@ -123,10 +155,162 @@ table.resultListing td, table.resultListing th {
...
+  }
+
+  .badge.authorisedrepresentative {
+    display: block;
+    margin-left: 0rem;
+    margin-top: 0.1rem;
+  }
+}
+
+.warning {
```
[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/SellerDashboard/SellerDashboard.scss]: removed

File: apps/marketplace/components/SellerDashboard/SellerDashboard.scss [hunk 5]
```diff
@@ -123,10 +155,162 @@ table.resultListing td, table.resultListing th {
...
+			'</svg>');
+
+		.ie8 &,
+		.lt-ie8 & {
+			content: '!';
+		}
+	}
+}
+
+.error {
```
[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/SellerDashboard/SellerDashboard.scss]: removed

File: apps/marketplace/components/SellerDashboard/SellerDashboard.scss [hunk 6]
```diff
@@ -123,10 +155,162 @@ table.resultListing td, table.resultListing th {
...
+				'<path fill="#313131" d="M17 8.4L15.6 7 12 10.6 8.4 7 7 8.4l3.6 3.6L7 15.6 8.4 17l3.6-3.6 3.6 3.6 1.4-1.4-3.6-3.6z"/>' +
+			'</svg>');
+
+		.ie8 &,
+		.lt-ie8 & {
+			content: '!';
+		}
+	}
+}
+.error {
```
[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/SellerDashboard/SellerDashboard.scss]: removed

File: apps/marketplace/components/SellerDashboard/SellerDashboard.scss [hunk 7]
```diff
@@ -123,10 +155,162 @@ table.resultListing td, table.resultListing th {
...
+  // background-color: red;
+}
+.error::before {
```
[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/SellerDashboard/SellerDashboard.scss]: removed

File: apps/marketplace/components/SellerDashboard/SellerDashboard.scss [hunk 8]
```diff
@@ -123,10 +155,162 @@ table.resultListing td, table.resultListing th {
...
+  top: 50%;
+  display: block;
+  width: 32px;
+  height: 32px;
+  background-repeat: no-repeat;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'%3E%3Cpath fill='%23ff635c' d='M0,0v24h24V0H0z M22.9,16.6L16.5,23H7.4L1,16.5V7.4L7.4,1h9.2l6.3,6.4V16.6z'/%3E%3C/svg%3E");
+  
+  
+}
+.error::after {
```
[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/SellerDashboard/SellerDashboard.scss]: removed

File: public/custom.css [hunk 9]
```diff
@@ -583,6 +583,29 @@ li:first-child.au-marketplace-header-link-list a {
   display: block;
 }
 
+.notification {
```
[the remark this answers is not in the document]
Comment (malmas18) [reply public/custom.css]: this doesn't work in the buyer front end

File: apps/marketplace/components/SellerDashboard/Team.js [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+                          </div>
+                        ) : (
+                          ''
+                        )}
+                      </td>
+                      <td className={styles.colAction}>
+                        {item.type === 'ar' ? (
+                          <a href="/sellers/edit/?step=your-info">Change representative</a>
+                        ) : (
+                          <a href="#remove" onClick={() => this.props.removeClicked(item)}>
```
Comment (tobiasbrown) [inline apps/marketplace/components/SellerDashboard/Team.js]: When this is clicked, the page alert isn't focused.  Can we fix this?

File: apps/marketplace/components/SellerDashboard/SellerDashboard.scss [hunk 11]
```diff
@@ -2,6 +2,24 @@
   margin-bottom: 100px;
 }
 
+.noRecords {
+  margin-top: 24px;
```
[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/SellerDashboard/SellerDashboard.scss]: recommended by design to add this spacing

File: apps/marketplace/components/SellerDashboard/Team.js (hunk 10 above)
Comment (malmas18) [reply apps/marketplace/components/SellerDashboard/Team.js]: fixed, cancel is focused now

**Pull request merged**